draft-sheffer-wimse-s2s-protocol.md

```diff
@@ -136,6 +136,89 @@ This document uses "service" and "workload" interchangeably. Otherwise, all term
 
 ## Option 2: Authentication Based on HTTP Message Signatures
 
+This option uses the WIMSE Identity Token (ref TBD) to sign the request and optionally, the response.
+This specification defines a profile of the Message Signatures specification {{!RFC9421}}.
+
+The request is signed as per {{RFC9421}}. The following derived components MUST be signed:
+
+* `@method`
+* `@request-target`
+
+In addition, the following headers MUST be signed when they exist:
+
+* `Content-Type`
+* `Content-Digest`
+* `Authorization`
+* `Txn-Token` {{?I-D.ietf-oauth-transaction-tokens}}
+* TBD that includes the WIT
```

Comment:
Do we envision cases where the WIT would not be included?

Reply:
@jsalowey No I don't, just waiting for the header definition from @bc-pi.